cached_network_image/pubspec.yaml

```diff
@@ -2,24 +2,26 @@ name: cached_network_image
 description: Flutter library to load and cache network images.
   Can also be used with placeholder and error widgets.
 homepage: https://github.com/Baseflow/flutter_cached_network_image
-version: 3.2.3
+topics:
+  - cache
+  - image
+  - network-image
+version: 3.2.4
```

Comment:
I think this could go to 3.3.0 as it's adding new features (such as the errorListener)